### PR TITLE
adding normative statement about source information

### LINKED ISSUE #537: Move old 'Sources and annotations' page content to 'Schema reference' page
**Summary of the bug or issue**

The [information here](https://standard.openownership.org/en/latest/schema/guidance/provenance.html#provenance) would sit better as part of the [Schema reference](https://standard.openownership.org/en/latest/schema/reference.html) page. (There may be normative requirements that would better sit in the schema too.)

### PR DESCRIPTION
**Pull request by kathryn-ods**
# Overview

- What does this pull request do? adds a normative requirement that statements should include source info 
- How can a reviewer test or examine your changes?
- Who is best placed to review it? 

Closes #537 

## Translations

- [ ] New or edited strings appearing as a result of this PR will be picked up for translation
- [ ] I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [ ] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
- [ ] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [ ] I've updated the changelog, if necessary.
- [ ] I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- [ ] I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md

### CODE REVIEW

**Comment by kathryn-ods** [comment]
@kd-ods imo this is the only thing in the sources and annotations page that isn't otherwise covered in the schema/docs. See comments in #537 for more detail.

File: schema/statement.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
           },
           "source": {
             "title": "Source",
-            "description": "The source of information in this statement.",
+            "description": "The source of information in this statement. Each statement SHOULD contain source information",
```
Comment (kd-ods) [inline schema/statement.json:114]:
```suggestion
            "description": "The source of information in this statement. Each statement SHOULD contain source information.",
```

**Review by kd-ods — CHANGES_REQUESTED** [review]
If we're deleting the provenance page, I feel we need to do something with the verification-related content... and also the point about using annotations to record data processing/cleaning/enrichment.

On the first, maybe just an edit to the `source.type` field description, to:

> "The types of the source, using the sourceType codelist. Include 'verified' in the array if the information in the Statement has undergone a verification process."

On the second, I actually think this is a bigger issue. If annotations are to be used to note data cleaning etc, then those reasons should be in the motivations codelist. BUT let's not get bogged down in that for this release. We can spin up a new issue with the title "Consider adding 'cleaning' and 'enrichment' to the motivations codelist" with some background and rationale. Then come back to it in the future.

What do you think?

**Comment by kathryn-ods** [comment]
@kd-ods have made those changes and made #638 for the annotations issue

**Review by kd-ods — APPROVED** [review]
Great. This is good to go.

**Pull request merged**